libvast/src/detail/add_message_types.cpp

```diff
@@ -19,6 +19,7 @@
 #include "vast/die.hpp"
 #include "vast/expression.hpp"
 #include "vast/operator.hpp"
+#include "vast/partition_synopsis.hpp"
```

Comment:
Why?

Reply:
We do use partition synopsis as a message type so this include was in fact missing. Not sure why it didn't error before; probably because the type used to be recursively included by some other header.